.travis.yml

```diff
@@ -49,6 +49,9 @@ before_install:
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
 
+before_script:
+  echo 'dmbstring.func_overload=7' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
I'd build out additional jobs in the matrix that set an env variable, and then, here, check if that env variable is set, and, if so, do the overloading. That way we test both with and without the overloading.